--- scripts/makepkg.sh.in.old	2009-02-09 16:21:18.000000000 -0500
+++ scripts/makepkg.sh.in	2009-02-09 16:21:37.000000000 -0500
@@ -1094,8 +1094,10 @@
 	#
 	if [ "$newpkgver" != "" ]; then
 		if [ "$newpkgver" != "$pkgver" ]; then
-			sed -i "s/^pkgver=[^ ]*/pkgver=$newpkgver/" "./$BUILDSCRIPT"
-			sed -i "s/^pkgrel=[^ ]*/pkgrel=1/" "./$BUILDSCRIPT"
+			sed -i.sed -e "s/^pkgver=[^ ]*/pkgver=$newpkgver/" "./$BUILDSCRIPT" &&
+			rm -f "./${BUILDSCRIPT}.sed"
+			sed -i.sed -e "s/^pkgrel=[^ ]*/pkgrel=1/" "./$BUILDSCRIPT" &&
+			rm -f "./${BUILDSCRIPT}.sed"
 			source "$BUILDSCRIPT"
 		fi
 	fi
