--- scripts/repo-add.sh.in.orig	2009-08-13 20:50:14.000000000 -0400
+++ scripts/repo-add.sh.in	2009-08-13 20:50:39.000000000 -0400
@@ -194,7 +194,7 @@
 	# read info from the zipped package
 	local line var val
 	for line in $(bsdtar -xOqf "$pkgfile" .PKGINFO |
-			grep -v '^#' | sed 's|\(\w*\)\s*=\s*\(.*\)|\1 \2|'); do
+			grep -v '^#' | sed 's|\(\w*\)\s* = \s*\(.*\)|\1 \2|'); do
 		# bash awesomeness here- var is always one word, val is everything else
 		var=${line%% *}
 		val=${line#* }
