--- scripts/repo-add.sh.in.orig	2009-02-09 00:45:52.000000000 -0500
+++ scripts/repo-add.sh.in	2009-02-09 00:46:09.000000000 -0500
@@ -153,7 +153,7 @@
 	# read info from the zipped package
 	local line
 	for line in $(bsdtar -xOf "$pkgfile" .PKGINFO | \
-		grep -v "^#" | sed 's|\(\w*\)\s*=\s*\(.*\)|\1="\2"|'); do
+		grep -v "^#" | sed 's|\(\w*\) *= *\(.*\)|\1="\2"|'); do
 		eval "$line"
 		case "$line" in
 			group=*)    _groups="$_groups$group\n" ;;
