--- a/src/cairo-quartz-font.c
+++ b/src/cairo-quartz-font.c
@@ -749,7 +749,7 @@ _cairo_quartz_scaled_font_get_cg_font_ref (cairo_scaled_font_t *abstract_font)
     return ffont->cgFont;
 }
 
-
+#ifndef __LP64__
 /*
  * compat with old ATSUI backend
  */
@@ -789,3 +789,4 @@ cairo_atsui_font_face_create_for_atsu_font_id (ATSUFontID font_id)
 {
     return cairo_quartz_font_face_create_for_atsu_font_id (font_id);
 }
+#endif
diff --git a/src/cairo-quartz.h b/src/cairo-quartz.h
index 769b29f..bf89cd9 100644
--- a/src/cairo-quartz.h
+++ b/src/cairo-quartz.h
@@ -66,8 +66,10 @@ cairo_quartz_surface_get_cg_context (cairo_surface_t *surface);
 cairo_public cairo_font_face_t *
 cairo_quartz_font_face_create_for_cgfont (CGFontRef font);
 
+#ifndef __LP64__
 cairo_public cairo_font_face_t *
 cairo_quartz_font_face_create_for_atsu_font_id (ATSUFontID font_id);
+#endif
 
 #endif /* CAIRO_HAS_QUARTZ_FONT */
