--- fc.orig	2009-02-12 13:31:44.000000000 -0500
+++ fc	2009-02-12 13:32:00.000000000 -0500
@@ -120,7 +120,7 @@
 OUTF=a.out
 OUTO=
 cOPT=1
-set -- `getopt acCD:gI:L:m:N:O:U:o:r:sSt:uw6 "$@"`
+set -- `getopt acCf:D:gI:L:m:N:O:U:o:r:sSt:uw6 "$@"`
 case $? in 0);; *) rc=$?; exit;; esac
 CPPFLAGS=${CPPFLAGS:-'-I/usr/local/include'}
 CFLAGSF2C=${CFLAGSF2C:-'-I/usr/local/include'}
@@ -146,6 +146,10 @@
 	-D)	CPPFLAGS="$CPPFLAGS -D$2"
 		shift 2
 		;;
+     
+    -f) CFLAGS="$CFLAGS -f$2"
+        shift 2
+        ;;
 
 	-g)	CFLAGS="$CFLAGS -g"
 		F2CFLAGS="$F2CFLAGS -g"
@@ -169,11 +173,19 @@
 		shift 2
 		;;
 
-	-O)	case $2 in 1) O=-O1;; 2) O=-O2;; 3) O=-O3;; *) O=-O;; esac
-		case $O in -O);; *) shift;; esac
+	-O)	case $2 in 
+            1) O=-O1
+               shift 2 ;;
+            2) O=-O2
+               shift 2 ;;
+            3) O=-O3 
+               shift 2 ;;
+            *) O=-O
+               shift ;;
+        esac
+#		case $0 in -O);; *) shift;; esac
 		CFLAGS="$CFLAGS $O"
 #		CFLAGS="$CFLAGS $O -Olimit 2000"
-		shift
 		;;
 
 	-r)	case $2 in 8) F2CFLAGS="$F2CFLAGS -r8";;
@@ -234,13 +246,21 @@
 
 	'')	echo $0: 'unexpected null argument'; exit 1;;
 
-	*)
+	-*)
 		echo "invalid parameter $1" 1>&2
 		shift
 		;;
+    *)
+        break
+        ;;
 	esac
 done
-shift
+
+if [ x"$1" == x"--" ]; then
+    shift
+fi
+
+
 case $cOPT in 0) case $OUTO in '');; *) CFLAGS="$CFLAGS -o $OUTO";; esac;; esac
 while
 	test -n "$1"
