--- Makefile.pre.in.orig	2008-10-21 12:48:37.000000000 -0400
+++ Makefile.pre.in	2009-02-10 05:37:11.000000000 -0500
@@ -421,6 +421,9 @@
 libpython$(VERSION).sl: $(LIBRARY_OBJS)
 	$(LDSHARED) $(LDFLAGS) -o $@ $(LIBRARY_OBJS) $(SHLIBS) $(LIBC) $(LIBM) $(LDLAST)
 
+libpython$(VERSION).dylib: $(LIBRARY_OBJS)
+	$(LDSHARED) -dynamiclib -install_name $(prefix)/lib/$@ -o $@ $(LIBRARY_OBJS) $(SHLIBS) $(LIBC) $(LIBM) $(LDLAST)
+
 # This rule is here for OPENSTEP/Rhapsody/MacOSX. It builds a temporary
 # minimal framework (not including the Lib directory and such) in the current
 # directory.
